lib/Runtime/Library/WebAssemblyMemory.h

```diff
@@ -9,6 +9,9 @@ namespace Js
 {
     class WebAssemblyMemory : public DynamicObject
     {
+    protected:
+        DEFINE_VTABLE_CTOR( WebAssemblyMemory, DynamicObject );
+        DEFINE_MARSHAL_OBJECT_TO_SCRIPT_CONTEXT( WebAssemblyMemory );
```

Comment:
When I tried to build nojit with vs2013 it complained that WebAssemblyMemory/Table didn't have a constructor.
I added this by looking at other javascript objects, but I don't know if it needs something else